dist/components/TopBar/TopBar.css

```diff
@@ -0,0 +1,45 @@
+@import '../../css/base.css';
+
+.top-bar {
+  width: 100%;
+  background-color: var(--g3-color__base-blue);
+  display: flex;
+  align-items: center;
+}
+
+.top-bar__header {
+  background-color: var(--g3-color__base-blue);
+  overflow: hidden;
+  vertical-align: middle;
+  margin: auto;
+  width: 1216px;
+}
+
+.top-bar__nav {
+  float: right;
+  color: var(--g3-color__white);
+  overflow: hidden;
+}
+
+.top-bar__link {
+  border-right: 2px solid #6896c6;
+  height: 20px;
+  display: inline-block;
+  text-align: center;
+  vertical-align: middle;
+  padding: 0 0;
+  margin-top: 10px;
+  margin-bottom: 10px;
+}
+
+.top-bar button {
```

Comment:
How about following BEM rule and renaming this to e.g. `.top-bar__button` or `.top-bar__user-link`